packages/scaffold-config/test/unit/detect.spec.ts

```diff
@@ -304,4 +304,16 @@ describe('detectLanguage', () => {
 
     expect(actual).to.eq('ts')
   })
+
+  it('ignores node_modules when checking for tsconfig.json', async () => {
```

Comment:
Can we add a check for this glob patter as well to have full coverage?
 https://github.com/cypress-io/cypress/blob/10.0-release/packages/scaffold-config/src/detect.ts#L135

Reply:
We've got a few tests for `ts` files inside of `cypress` already, for example: https://github.com/cypress-io/cypress/blob/10.0-release/packages/scaffold-config/test/unit/detect.spec.ts#L255-L263